apps/web/src/components/layouts/AppLayout/Header/index.tsx

```diff
@@ -80,30 +86,65 @@ export type AppHeaderProps = {
   navigationLinks: INavigationLink[]
   currentUser: SessionUser | undefined
   breadcrumbs?: IBreadCrumb[]
+  sectionLinks?: INavigationLink[]
 }
 export default function AppHeader({
   breadcrumbs = [],
   navigationLinks,
   currentUser,
+  sectionLinks,
 }: AppHeaderProps) {
   const info = currentUser ? getUserInfoFromSession(currentUser) : null
   return (
-    <header className='sticky top-0 px-6 py-3 flex flex-row items-center justify-between border-b border-b-border'>
-      <Breadcrump breadcrumbs={breadcrumbs} />
-      <div className='flex flex-row items-center gap-x-6'>
-        <nav className='flex flex-row gap-x-2'>
-          {navigationLinks.map((link, idx) => (
-            <NavLink key={idx} {...link} />
-          ))}
-        </nav>
-        {info ? (
-          <Avatar
-            alt={info.name}
-            fallback={info.fallback}
-            className='w-6 h-6'
-          />
-        ) : null}
+    <header className='sticky top-0 flex flex-col border-b border-b-border'>
+      <div className='px-6 py-3 flex flex-row items-center justify-between '>
```

Comment:
Please think in terms of the design system. Do the component as a design system elemen. Tabs are clearly one

Reply:
Not sure what could be a design system component in this PR tbh

Reply:
Can you show a screenshot?

Reply:
![image](https://github.com/user-attachments/assets/b32e8362-9c22-498c-a9ba-8f894b5ffe78)

Reply:
but it depends on nextjs' LInk component so it can't go in web-ui (no, i'm not gonna pass an onclick function with a programatic navigate)

Reply:
It can

```javascript
const tabs = useMemo(() => ([
   { 
     label: 'Projects',
     selected: currentPath === 'bla-bla',
     onClick: () => router.push(ROUTES.projects.root)
   },
   { 
     label: 'Evaluations',
     selected: currentPath === 'bla-bla',
     onClick: () => router.push(ROUTES.evaluations.root)
   },
   { 
     label: 'Settings',
     selected: currentPath === 'bla-bla',
     onClick: () => router.push(ROUTES.settins.root)
   },
]), [currentPath])

return (
  <Tabs tabs={tabs} />
)
```

Reply:
> (no, i'm not gonna pass an onclick function with a programatic navigate)